src/runtime/lpython/lpython.py

```diff
@@ -299,6 +299,31 @@ def __call__(self, *args, **kwargs):
             else:
                 new_args.append(arg)
         return self.cf(*new_args)
+    
+class jit:
+    def __init__(self, func):
+        self.func = func
+
+    def __call__(self, *args, **kwargs):
+        # Get the source code of the function
```

Comment:
This will be used by just CPython tests and has no relation with LPython's LLVM backend. Can CPython not ignore the JIT and return a regular function? As we will need to handle `lpython.jit` at ASR or LLVM backend.

Reply:
I don't know. I was just trying to follow the steps from what @certik wrote.

Reply:
It looks like this does steps 1. and 2. from https://github.com/lcompilers/lpython/issues/703#issuecomment-1478344669.

Now we need to do step 3.